### PR TITLE
feat: use date pickers from date-components

### LINKED ISSUE #894: Use date pickers from https://github.com/gravity-ui/date-components
in "Top shards" and "Top queries" tab.
Check if it is somewhere else...

### PR DESCRIPTION
**Pull request by astandrik**
Closes #894

Stand: [nda.ya.ru/t/F3W_HVqz773RNr](https://nda.ya.ru/t/F3W_HVqz773RNr)

### CODE REVIEW

File: src/components/DateRange/DateRange.tsx [hunk 1]
```diff
@@ -28,8 +28,14 @@ export const DateRange = ({from, to, className, onChange}: DateRangeProps) => {
     const handleUpdate = React.useCallback<NonNullable<RelativeRangeDatePickerProps['onUpdate']>>(
         (pickerValue) =>
             onChange?.({
-                from: dateTimeParse(pickerValue?.start?.value)?.valueOf(),
-                to: dateTimeParse(pickerValue?.end?.value)?.valueOf(),
+                from:
+                    pickerValue?.start?.type === 'relative'
```
Comment (Raubzeug) [inline src/components/DateRange/DateRange.tsx:32]: Lets add a helper like
```
function getValue({type, value}) {
if (type === 'relative') ....
}
```

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/components/DateRange/DateRange.tsx (hunk 1 above)
Comment (astandrik) [reply src/components/DateRange/DateRange.tsx:32]: fixed

**Comment by astandrik** [comment]
@Raubzeug Всё готово и лежит на стенде)

File: src/components/DateRange/i18n/ru.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "date-format": "DD.MM.YYYY",
```
Comment (Raubzeug) [inline src/components/DateRange/i18n/ru.json:2]: We decided not to translate application to Russian. So, it's no need to add RU keysets.
And if you are editing keysets where ru presents, please remove it.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/components/DateRange/i18n/ru.json (hunk 2 above)
Comment (astandrik) [reply src/components/DateRange/i18n/ru.json]: fixed

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**